### PR TITLE
Rename net income retained earnings adjustment column

### PR DESCRIPTION
**Pull request by fivetran-avinash**
**Are you a current Fivetran customer?** 

**What change(s) does this PR introduce?** 

Modified `int_quickbooks__retained_earnings` intermediate model to more accurately reflect retained earnings accounts as "Net Income Adjustment". 

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.) 
The change is only adding modifying a field.

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [ ] Yes (This is being merged into a non-main branch, where we will update the `dbt_project` versions when it's ready to merge. 

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature [#57]
- [ ] No 

**How did you test the PR changes?** 

 

- [x] BuildKite  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:apple:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: dbt_project.yml [hunk 1]
```diff
@@ -9,11 +9,11 @@ models:
...
     +schema: quickbooks
     double_entry_transactions:
-      +materialized: ephemeral
+      +materialized: table
     transaction_lines:
-      +materialized: ephemeral
+      +materialized: table
     intermediate:
-      +materialized: ephemeral
+      +materialized: table
```
Comment (fivetran-joemarkiewicz) [inline dbt_project.yml:16]: Should these be changed back?

**Review by fivetran-joemarkiewicz — APPROVED** [review]
I have just a few small comments and questions. Once they are applied we this looks good to me!

I do think the customer has an additional ask in generating the previous periods net income in the form of a true retained earnings field. That will be a bigger lift and can exist outside of this PR and task.

File: dbt_project.yml (hunk 1 above)
Comment (fivetran-avinash) [reply dbt_project.yml:16]: Yes. Thanks for catching!

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,7 @@
 - Added `class_id` to `quickbooks__general_ledger`, `quickbooks_general_ledger_by_period`, and `quickbooks__balance_sheet`; add in class values for all intermediate models necessary to pass into final models. [#58](https://github.com/fivetran/dbt_quickbooks/pull/58).
 - Added `source_relation` field to all Quickbooks models to allow customers, if they have multiple Quickbooks connectors, to union them inside the package. [#62](https://github.com/fivetran/dbt_quickbooks/pull/62).
 - Added tests to all final models, particularly to test uniqueness across a combination of columns, including `source_relation`. 
+- Modified `int_quickbooks__retained_earnings` intermediate model to more accurately reflect retained earnings accounts as "Net Income Adjustment". [#66](https://github.com/fivetran/dbt_quickbooks/pull/66)
```
[the remark this answers is not in the document]
Comment (fivetran-avinash) [reply CHANGELOG.md]: Done!

**Pull request merged**